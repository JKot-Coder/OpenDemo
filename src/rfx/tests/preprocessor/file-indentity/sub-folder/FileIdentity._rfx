// :RUN_RFXC -P %STD_OUTPUT% %INPUT%

#include "../b.hrfx"
#include "../c.hrfx"

#include "./../b.hrfx"

float test(float x)
{
	return foo(x) + bar(x);
}