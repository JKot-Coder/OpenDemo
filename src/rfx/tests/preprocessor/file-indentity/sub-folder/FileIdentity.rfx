#include "../b.hrfx"
#include "../c.hrfx"

#include "./../b.hrfx"

float test(float x)
{
	return foo(x) + bar(x);
}