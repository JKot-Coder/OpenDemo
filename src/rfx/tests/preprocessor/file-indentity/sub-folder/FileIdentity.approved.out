PreprocessorOutput:[
	float bar(float x) { return x; }
	float foo(float x) { return x; }
	float test(float x)
	{
	    return foo(x) + bar(x);
	}
]
Diagnostic:[
]
